lib/Scheduler/graph_trans.cpp

```diff
@@ -2,11 +2,71 @@
 #include "opt-sched/Scheduler/bit_vector.h"
 #include "opt-sched/Scheduler/logger.h"
 #include "opt-sched/Scheduler/register.h"
+#include "llvm/ADT/ArrayRef.h"
 #include "llvm/ADT/STLExtras.h"
+#include "llvm/ADT/SmallVector.h"
+#include <algorithm>
+#include <vector>
 #include <list>
 
 using namespace llvm::opt_sched;
 
+namespace {
+// Gets the registers which may be lengthened by scheduling nodeB after nodeA,
+// as compared to if nodeB was scheduled before nodeA
+llvm::SmallVector<Register const *, 10> possiblyLengthenedIfAfterOther(
```

Comment:
Should be consistent with where const goes in ptr to const types. 
`const Register *`

Reply:
Good point. I'm going to constantly forget this; I do east const most of the time. I'll change this, and I'll see if I can find a tool for myself to catch these errors automatically